### ISSUE TITLE
🐛 BUG: Can't format astro files

### ISSUE THREAD

**Issue by altano** [withastro/prettier-plugin-astro#276]
### Describe the Bug

I can't format `.astro` files using [the recommended settings](https://github.com/withastro/prettier-plugin-astro#using-in-vs-code). I get an error in Prettier.

# Steps to reproduce
see below

# Workaround
1. Open VS Code command palette (ctrl+shift+p)
2. "Format Document With..."
3. Select `Astro`
4. Document formats correctly, no errors:
````
---
---

<div>
  <p>Hi</p>
  <p>Hi2</p>
</div>
```

### Steps to Reproduce

1. Add the recommended settings in `.vscode/settings.json`:
```json
  "prettier.documentSelectors": ["**/*.astro"],
  "[astro]": {
    "editor.defaultFormatter": "esbenp.prettier-vscode"
  },
```
2. Installed extensions:
```
esbenp.prettier-vscode
astro-build.astro-vscode
```
3. Create unformatted `Test.astro`:
```astro
---

---

<div>
  <p>Hi</p>
    <p>Hi2</p>
</div>
```
4. Save file to trigger format. Format fails with error `Error formatting document. Error: Go program has already exited`
<details>
<summary>Error log from `Prettier`</summary>

```
["INFO" - 10:52:50 PM] Formatting file:///home/alan/src/<redacted>/src/components/home/Test.astro
["INFO" - 10:52:50 PM] Using config file at '/home/alan/src/<redacted>/.prettierrc'
["INFO" - 10:52:50 PM] Using ignore file (if present) at /home/alan/src/<redacted>/.prettierignore
["INFO" - 10:52:50 PM] File Info:
{
  "ignored": false,
  "inferredParser": "astro"
}
["INFO" - 10:52:50 PM] Detected local configuration (i.e. .prettierrc or .editorconfig), VS Code configuration will not be used
["INFO" - 10:52:50 PM] Prettier Options:
{
  "filepath": "/home/alan/src/<redacted>/src/components/home/Test.astro",
  "parser": "astro",
  "useTabs": false,
  "tabWidth": 2,
  "endOfLine": "lf",
  "trailingComma": "es5",
  "bracketSpacing": false,
  "singleQuote": false,
  "astroAllowShorthand": false
}
["ERROR" - 10:52:50 PM] Error formatting document.
Error: Go program has already exited
    at Go._resume (file:///home/alan/src/<redacted>/node_modules/@astrojs/compiler/node/wasm_exec.js:435:19)
    at Object.parse (file:///home/alan/src/<redacted>/node_modules/@astrojs/compiler/node/wasm_exec.js:447:16)
    at Promise.then (file:///home/alan/src/<redacted>/node_modules/@astrojs/compiler/node/index.js:54:78)
    at new Promise (<anonymous>)
    at Object.parse (file:///home/alan/src/<redacted>/node_modules/@astrojs/compiler/node/index.js:54:36)
    at file:///home/alan/src/<redacted>/node_modules/@astrojs/compiler/node/index.js:8:51
["INFO" - 10:52:50 PM] Formatting completed in 17ms.
```

</details>

**Comment by antonyfaris** [comment]
That's odd, I can't reproduce the error.

**Comment by altano** [comment]
Interesting. Let me jot down some more details to help narrow this down:

* VS Code Extensions:
  * astro-build.astro-vscode@0.23.1 (latest)
  * esbenp.prettier-vscode@9.8.0 (latest)
* npm packages:
  * astro@1.1.5 (latest)
  * prettier@2.7.1 (latest)
  * prettier-plugin-astro@0.5.4 (latest)
* vscode@1.70.2 (**NOT** latest)

.prettierrc:
```json
{
  "trailingComma": "es5",
  "tabWidth": 2,
  "bracketSpacing": false,
  "singleQuote": false,
  "astroAllowShorthand": false
}
```

I updated vscode to 1.71 (latest) just to see if that would help and the problem went away. I'm not sure what to make of that: I doubt 1.70.2 had a bug that was fixed in 1.71: more likely the update just unstuck something that was busted. I'll close this out and if someone else reproduces this issue I guess they can reopen it.

Thanks for taking a look, @antonyfaris
